### PR TITLE
Add Emergency Status fields to pierone describe

### PR DESCRIPTION
**Pull request by jmcs**
- Add Emergency Status fields to pierone describe
- Automata details box generation
- Improve tests

### CODE REVIEW

File: pierone/cli.py [hunk 1]
```diff
@@ -264,41 +263,44 @@ def describe(config, team, artifact, tag, url):
...
-    click.echo("Checker Status         ┃ {}".format(checker_status))
-    click.echo("Checker Status Date    ┃ {checker_status_received_at}".format_map(tag_info))
-    click.echo("Checker Status Reason  ┃ {checker_status_reason}".format_map(tag_info))
-    click.echo("Checker Status Details ┃ {}".format(status_details.pop(0) if status_details else ""))
-    for line in status_details:
-        click.echo("                       ┃ {}".format(line))
-    if tag_info["user_status"]:
+        details_box.set("Compliance Information", "Valid SCM Source", "No SCM Source")
+    details_box.set("Compliance Information", "Effective Status", effective_status)
+    details_box.set("Compliance Information", "Checker Status", checker_status)
```
Comment (lotharschulz) [inline pierone/cli.py]: What about  `Compliance Checker Status` ?

**Comment by lotharschulz** [comment]
👍

File: pierone/cli.py (hunk 1 above)
Comment (jmcs) [reply pierone/cli.py]: Checker Status is the Compliance Checker Status

**Comment by marcinzaremba** [comment]
👍

**Pull request merged**